helm/charts/keto/values.yaml

```diff
@@ -38,31 +38,60 @@ securityContext:
   allowPrivilegeEscalation: false
   privileged: false
 
+ingress:
+  read:
+    enabled: false
+    annotations: {}
+    # kubernetes.io/ingress.class: nginx
+    # kubernetes.io/tls-acme: "true"
+    hosts:
+      - host: chart-example.local
+        paths: []
+    tls: []
+    #  - secretName: chart-example-tls
+    #    hosts:
+    #      - chart-example.local
+  write:
+    enabled: false
+    annotations: {}
+    # kubernetes.io/ingress.class: nginx
+    # kubernetes.io/tls-acme: "true"
+    hosts:
+      - host: chart-example.local
+        paths: []
+    tls: []
+    #  - secretName: chart-example-tls
+    #    hosts:
+    #      - chart-example.local
+
```

Comment:
Same host and port for both endpoints? I guess this is just an example but maybe still change that to not confuse people.

Reply:
Or is it possible to have it on the same?

Reply:
It is possible by setting specific paths, but you're right I will make the example more verbose